src/main/java/com/github/javafaker/Internet.java

```diff
@@ -297,4 +297,47 @@ public String uuid() {
     private <T> T random(T[] src) {
         return src[faker.random().nextInt(src.length)];
     }
+
+    public String userAgent(USER_AGENT userAgent){
```

Comment:
could you add a space between the closing brace and the opening parenthesis?

```
userAgent) {
```
add apply that to all the added methods.. sorry I really should add some linting so it's easier for contributors